http/http.go

```diff
@@ -111,9 +111,12 @@ func (hct *httpClientTracer) finishSpan() {
 // this function - probably a static string for each callsite
 // you can disable compression with compress=false for endpoints that don't
 // support it
-func PostHelper(ctx context.Context, httpClient *http.Client, tc *trace.Client, method string, endpoint string, bodyObject interface{}, action string, compress bool, log *logrus.Logger) error {
+func PostHelper(ctx context.Context, httpClient *http.Client, tc *trace.Client, method string, endpoint string, bodyObject interface{}, action string, compress bool, extraTags map[string]string, log *logrus.Logger) error {
 	span, _ := trace.StartSpanFromContext(ctx, "")
 	span.SetTag("action", action)
+	for k, v := range extraTags {
+		span.SetTag(k, v)
```

Comment:
Sadly, this won't assign the tag to the error metric to the metrics attached to the span - you'll have to merge those manually & ugh.

Reply:
oh darn, that doesn't just work? should i.... make that work?

Reply:
I think that'll be difficult & mess up the metrics' tags that we attach to spans... so I think locally fixing would be better /:

Reply:
haha, yeah, that's why i asked. alright i'll merge the tags manually